SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -79,13 +76,81 @@ bool CheckClassNameListValidity(const TArray<UClass*>& Classes)
 }
 }// ::
 
-void GenerateSchemaFromClasses(const TArray<UClass*>& Classes, const FString& CombinedSchemaPath)
+bool CheckVariableNameValidity(TSharedPtr<FUnrealType> TypeInfo)
```

Comment:
```suggestion
bool CheckIdentifierNameValidity(TSharedPtr<FUnrealType> TypeInfo)
```
(or something like this, since it's not only variables)